docs/user_guide/1_4_Installing_the_software.md

```diff
@@ -0,0 +1,113 @@
+
+```eval_rst
+.. prompt:: bash $
+
+   sudo dmesg
+```
+command in the console. At the bottom, there will be information similar to the one below:
+
+```bash
+[490.757853] IPv6: ADDRCONF(NETDEV_CHANGE): enp0s25: link becomes ready
+```
+In the above example, ‘enp0s25’ is the interface ID that is needed.
+
+## Get ROS Upload login credentials
+
+If you want to upload technical logged data (ROS logs, backtraces, crash dumps etc.) to our server and notify the Shadow's software team to investigate your bug then you need to enable logs uploading in the one-liner. In order to use this option you need to obtain a unique upload key by emailing sysadmin@shadowrobot.com. When you receive the key you can use it when running the one-liner installation tool. To enable the logs uploading you need to add the command line option ```-ck true``` to the one-liner.
+After executing the one-liner, it will prompt you to enter your upload key and press enter to continue. Pleaser copy and paste your key from the email you received by Shadow Robot.
+
+## Check your hand configuration branch
+
+You should have the name of your [sr_config](https://github.com/shadow-robot/sr-config) hand branch which contains the specific configuration of your hand (calibration, controller tuning etc…).
+Usually it is something like this: ``shadowrobot_XXXXXX``. Where XXXXXX are the 6 digits contained in the serial number of the hand labelled underneath the robot base.
+
+If you are unsure please contact us.
+
+## Run the one-liner
+
+The one-liner will install Docker, pull the image from Docker Hub, and create and run a container with the parameters specified. In order to use it, use the following command:
+
+```eval_rst
+.. Note:: Please remember to replace [EtherCAT interface ID] with your Interface ID and [sr_config_branch] with your unique sr_config branch
+```
+
+ROS Kinetic (Recommended):
+
+```eval_rst
+.. prompt:: bash $
+
+    bash <(curl -Ls http://bit.ly/launch-sh) -i shadowrobot/dexterous-hand:kinetic-release -n arm-and-hand -sn Arm_Hand_Launcher -e [EtherCAT interface ID] -b [sr_config_branch] -l false
+```
+
+Examples:
+For Interface ID ```ens0s25``` and sr_config_branch ```shadow_12345```
+
+```eval_rst
+.. prompt:: bash $
+
+    bash <(curl -Ls http://bit.ly/launch-sh) -i shadowrobot/dexterous-hand:kinetic-release -n arm-and-hand -sn Arm_Hand_Launcher -e ens0s25 -b shadow_12345 -l false
+```  
+
+Same as above but with ROS logs upload enabled
+
+```eval_rst
+.. prompt:: bash $
+
+    bash <(curl -Ls http://bit.ly/launch-sh) -i shadowrobot/dexterous-hand:kinetic-release -n arm-and-hand -sn Hand_Launcher -e ens0s25 -b shadow_12345  -l false -ck true 
+```  
+
+If you have an Nvidia graphics card, you can add -nv to set the nvidia-docker version. Use ``-nv 1`` or ``-nv 2`` for version 1.0 or 2.0 respectively. Must be used with ``-g true``.
+
+You can also add -r true in case you want to reinstall the docker image and container. When it finishes it will show:
```

Comment:
```suggestion
You can also add -r true in case you want to reinstall the docker image and container. 

When the one-liner finishes it will show:
```